app/views/users/show.html.erb

```diff
@@ -1,2 +1,28 @@
-<h1>The User</h1>
 
+<div class="user_page">
+	<div class="user">
+	
+		<%= image_tag(@user.photo, alt: "user photo", class: "user_photo") %>
+	
+		<div class="user_info">
+			<h2>
+			<%= link_to @user.name, user_path(@user.id) %>
+			</h2>		
+			<p>Number of posts: <%= @user.posts_counter %></p>
+		</div>
+	</div>
+		
+	<div class="user_bio">
+		<h4>Bio</h4>
+		<article> <%= @user.bio %> </article>
+	</div>
+	
+	<ul class="posts_list">
+	<% @posts.each do |post| %>
+		<%= render partial: "users/post", locals: { post: post } %>
+	<% end %>
+	</ul>
+	
+	<button><%= link_to "See all posts", user_posts_path(@user.id) %></button>
+	
+</div>
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you properly indent your code.

Reply:
Thanks for the notice I'll work on it